CONTRIBUTING.md

```diff
@@ -94,6 +90,11 @@ Instead, if there are some **obsolete snapshots**, you can `UPDATE_SNAPS=clean`:
 UPDATE_SNAPS=clean go test -v ./internal/...
 ```
 
+
+### Adding new tests
+
+Adding tests for the tokenizer, scanner, and printer can be found in `internal/token_test.go`, `internal/js_scanner_test.go`, and `internal/printer/printer_test.go`, respectively.
```

Comment:
I don't really get how this explain how to make a new snapshot test, perhaps an example would be great?

Reply:
The commit [`037482d` (#1011)](https://github.com/withastro/compiler/pull/1011/commits/037482d117c52db47a3e1c5882a2e564bfcf64d4) adds more explanation